keep-ui/app/rules/CorrelationTable.tsx

```diff
@@ -94,14 +95,6 @@ export const CorrelationTable = ({ rules }: CorrelationTableProps) => {
       columnHelper.accessor("name", {
         header: "Correlation Name",
       }),
-      columnHelper.display({
```

Comment:
why? how a user would have indication for if this rule works or not?

Reply:
These columns were always empty. I'm going to improve this UI in one of next steps

Reply:
https://github.com/keephq/keep/issues/1664